kodiak/config_utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import Union
+
+import pydantic
+import toml
+from markupsafe import escape
+
+
+def get_markdown_for_config(
+    error: Union[pydantic.ValidationError, toml.TomlDecodeError],
+    config_str: str,
+    git_path: str,
+) -> str:
+    config_escaped = escape(config_str)
```

Comment:
I think it would be more clear if we just did `import markupsafe` and called `markupsafe.escape(`